x/incentives/keeper/distribute_test.go

```diff
@@ -394,6 +357,186 @@ func (s *KeeperTestSuite) TestDistributeToConcentratedLiquidityPools() {
 	}
 }
 
+// TestDistribute_ExternalIncentives_NoLock tests the distribution of externally
+// created NoLock gauges. It creates an external gauge with the correct configuration
+// and uses it to attempt to distribute tokens to a concentrated liquidity pool.
+// It attempts to distribute with all possible gauge configurations and with various tokens.
+// However, it does not test distribution of NoLock gauges.
+func (s *KeeperTestSuite) TestDistribute_ExternalIncentives_NoLock() {
+	const (
+		defaultCLPool       = uint64(1)
+		defaultBalancerPool = uint64(2)
+
+		defaultAmount = int64(5000)
+	)
+
+	fiveKRewardCoins := sdk.NewInt64Coin(defaultRewardDenom, defaultAmount)
+	tenKOtherCoin := sdk.NewInt64Coin(otherDenom, defaultAmount+defaultAmount)
+
+	defaultBothCoins := sdk.NewCoins(fiveKRewardCoins, tenKOtherCoin)
+
+	defauBlockTime := time.Unix(123456789, 0)
+	oneHourAfterDefault := defauBlockTime.Add(time.Hour)
+
+	type test struct {
+		// setup
+		isPerpertual       bool
+		tokensToAddToGauge sdk.Coins
+		gaugeStartTime     time.Time
+		gaugeCoins         sdk.Coins
+		distrTo            lockuptypes.QueryCondition
+		startTime          time.Time
+		numEpochsPaidOver  uint64
+		poolId             uint64
+
+		isBalancerPool bool
```

Comment:
seems we dont use this in the test either (isBalancerPool)

Reply:
done